features/admin.userstores.v1/components/user-stores-list.tsx

```diff
@@ -301,8 +310,8 @@ export const UserStoresList: FunctionComponent<UserStoresListPropsInterface> = (
         return null;
     };
 
-    const handleUserstoreEdit = (userstoreId: string) => {
-        if (userstoresConfig.onUserstoreEdit(userstoreId)) {
+    const handleUserstoreEdit = (userstoreId: string, typeName: string) => {
+        if (userstoresConfig.onUserstoreEdit(userstoreId) && (typeName !== "WSOutboundUserStoreManager")) {
```

Comment:
Shall we get the `WSOutboundUserStoreManager` from a constant?

Reply:
Improved with 6b183b654a74dcf0ef25292a34d6ec9e2e620d04.